metadata-foundationdb/metadata-foundationdb.c

```diff
@@ -108,9 +110,9 @@ void metadata_persist(FDBDatabase *database, const Metadata *metadata)
  * \return the Metadata stored in FoundationDB
  * \retval NULL if the MetaData could not be found or if there was an error unpacking
  */
-Metadata *metadata_get(FDBDatabase *database)
+Metadata *load_metadata()
```

Comment:
maybe this should be called `metadata_load` to follow the same pattern of `<object>_<verb>` as with `metadata_persist` above

Reply:
since the module is explicitly tied to metadata, we could get rid of the first part, and just leave the verb.